website/blog/2023-08-25-silver.md

```diff
@@ -0,0 +1,399 @@
+working with Java sources easier, though Metals will not strive to be a full Java language server.
+
+![java-completion](https://i.imgur.com/wLlh7ig.gif)
+
+## Liveness monitor
+
+Up to the current release, users would not be informed if something happened
+and the build server stopped responding. Thanks to [kasiaMarek](https://github.com/kasiaMarek) Metals will now monitor the state 
+of the build server and notify the user if it becomes unresponsive.
+
+![not-responsive](https://imgur.com/R2T5x3t.png)
+
+## Improvements for enums
+
+This release also brings in multiple fixes connected to enums: 
+
+- better highlighting on enum cases
+- improved navigation on enum parameters
+- multiple fixes on enum type parameters
+
+This great work was done by [kasiaMarek](https://github.com/kasiaMarek)
+
+![enum](https://i.imgur.com/tZrSDYC.gif)
+
+## Semantic highlighting
+
+Semantic highlighting enables syntax highlighting based on knowledge from the compiler.
+Thanks to this, users can have specific colors for parameters, methods, mutable `var`s, packages etc. 
+For the last half year, semantic highlighting was hidden behind a flag, but now we think it's ready 
+to be turned on by default. You can always change this using `metals.enableSemanticHighlighting` setting. 
+Some editors also require enabling this feature in other places. 
+In VS Code, you also need to set `editor.semanticHighlighting.enabled` to `true`.
+
+More details can be found [here](https://scalameta.org/metals/blog/2023/03/02/aluminium#introduce-support-for-semantic-tokens)
+
+## Miscellaneous
+
+- improvement: offer amend to handle single file scala-cli config on new file
+  ([kasiaMarek](https://github.com/kasiaMarek))
+- bugfix: Go to definition for named constructor params
+  ([jkciesluk](https://github.com/jkciesluk))
+- improvement: Add date and time to reports and backup log files
+  ([jkciesluk](https://github.com/jkciesluk))
+- bugfix: Semantic tokens misaligned in triple quote s-string
+  ([jkciesluk](https://github.com/jkciesluk))
+- bugfix: Completion item resolve for symbolic names
+  ([jkciesluk](https://github.com/jkciesluk))
+- bugfix: suggest correct arg name completions for lambda expressions
+  ([kasiaMarek](https://github.com/kasiaMarek))
+- bugfix: don't decode nor encode URIs
+  ([adpi2](https://github.com/adpi2))
+- feature: Add hover on class,object,type def in Scala 2
+  ([jkciesluk](https://github.com/jkciesluk))
+- bugfix: Don't report tokenizer errors as stale
+  ([tgodzik](https://github.com/tgodzik))
+- bugfix: Support Java when using sbt server via BSP
+  ([tgodzik](https://github.com/tgodzik))
+- improvement: Add proper Stable Presentation Compiler Support
+  ([rochala](https://github.com/rochala))
+- docs: Add info about the versioning strategy
+  ([tgodzik](https://github.com/tgodzik))
+- bugfix: do not emit cases in  pattern match as enum cases
+  ([kasiaMarek](https://github.com/kasiaMarek))
+- bugfix: accommodate junit encoded test names
```

Comment:
This adds a field to `TestCaseEntry`, which is part of the client API. It's a minor fix and the old API can be still used but clients might want to adjust. Not sure where such information should be put.

Reply:
We could have a section for plugin author or just add a paragraph with `**Plugin authors**`